studies/migrations/0100_alter_response_recording_method.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.25 on 2024-11-12 18:30
+
+from django.db import migrations, models
+
+EFP = 1
+EXTERNAL = 2
+JSPSYCH = 3
+
+
+def populate_recording_method(apps, schema_editor):
+    responses = apps.get_model("studies", "Response").objects.filter(
+        recording_method=None
+    )
+    for response in responses:
+        if response.study_type.id == EXTERNAL:
+            response.recording_method = None
```

Comment:
I'm fine with using None/NULL for responses from external studies here. But I just realized that the external responses will go back to being "pipe" after this migration, because of the default. So this will produce an inconsistency. Does that matter? Should we explicitly set the recording method to NULL for external study responses?

Reply:
Yes, we should.  I make that change.